extension/packagemanagerextension/nbextension/common.js

```diff
@@ -0,0 +1,175 @@
+// https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/endsWith
```

Comment:
Please delete anything that is not used.

Reply:
Sure